### PR TITLE
Defaults for Clariden

### PR DESCRIPTION
**Pull request by antonschafer**
Default config that allows for easy downloading on clariden with the slurm distributor and with the desired video format

### CODE REVIEW

File: video2dataset/configs/clariden.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    CutDetectionSubsampler: # from HD-VILA example
+        args:
+            cut_detection_mode: "all"
+            framerates: null
+            threshold: 11.5
+            min_scene_len: 15
+
+reading:
+    yt_args:
+        download_size: 720 # 360?
```
Comment (antonschafer) [inline video2dataset/configs/clariden.yaml:5]: We agreed on 720, right?

File: video2dataset/configs/clariden.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+subsampling:
+    CutDetectionSubsampler: # from HD-VILA example
```
Comment (antonschafer) [inline video2dataset/configs/clariden.yaml]: Might need to review these, just the defaults now

**Comment by antonschafer** [comment]
Right now, the slurm distributor allocates n_nodes entire nodes (see [code](https://github.com/swiss-ai/video2dataset/blob/a37d575f3ce91d21b842528586f4a311a1ea10bc/video2dataset/distributor.py#L210-L214)), so I set the number of cores to 128 in order not to leave any idle. Allocating entire nodes might be wasteful though, we should check if there's a better option

**Comment by kdu4108** [comment]
Great, thanks! Would you also be able to add the python script (maybe in dataset_examples?) you use to kick off the slurm job so we have that consistent too? Perhaps with some argparsing for the name/location of the dataset file, if it's not too much work and you can commit it today :)

**Comment by antonschafer** [comment]
The python script I used was the equivalent of this command (here for hd_vila). This should do the job. Maybe we should add a readme that outlines the steps for each dataset we processed. I'll add one for hd_vila

`video2dataset --url_list="hd_vila.parquet" --input_format="parquet" --output_format="webdataset" --output_folder="./outputs" --url_col="url" --encode_formats="{'video': 'mp4', 'audio':'m4a'}" --config="config.yaml"`

**Comment by kdu4108** [comment]
Yeah that works perfect, thanks!

**Comment by antonschafer** [comment]
Updated the default Clariden config. The cut detection lead to significant slowdowns (>2x at 360p, >10x at 720p). Also increased the number of threads per process to 16 instead of 4 which performed marginally better.

Now it's downloading the first 1k HD-vila videos in 720p (80GB) in 2 mins. So all of HD-VILA would take ~100h to download (and ~240TB).

If we need the cut detection we can run it afterwards on the unified dataset and look into speeding up, potentially via GPU

**Pull request merged**